heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -157,8 +165,12 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     // clean up the state of the topology in state manager
     cleanState(topologyName, Runtime.schedulerStateManagerAdaptor(runtime));
 
-    // Clean the connection when we are done.
-    LOG.fine("Scheduler killed topology successfully.");
+    if (hasStaleExecutionData) {
+      LOG.warning(String.format("Topology %s data is not found, but scheduler has tried to"
+          + " clean up data for it anyway.", topologyName));
+    } else {
+      LOG.fine(String.format("Scheduler killed topology %s successfully.", topologyName));
```

Comment:
this message might not be misleading - since we are not killing the topology I think.